src/components/ContentDecoder.tsx

```diff
@@ -67,6 +66,20 @@ export function ContentDecoder({ to, chainId = '1', descriptionResult, analyzeRe
                         <Risk key={id} risk={risk} />
                     ))}
                 </Collapsable>
+
+                {url && (
```

Comment:
Is there a better way to distinguish between wallet popup and manual search other than checking 'url' ?